addOns/exim/src/main/javahelp/help/contents/exim.html

```diff
@@ -65,10 +65,15 @@ <H2>Export URLs for Context</H2>
 <H1>ZAP API</H1>
 This add-on also exposes various ZAP API endpoints to facilitate programmatic use of the functionality.
 <ul>
-  <li><code>/exim/importHar (filePath*)</code></li>
-  <li><code>/exim/importModsec2Logs (filePath*)</code></li>
-  <li><code>/exim/importUrls (filePath*)</code></li>
-  <li><code>/exim/importZapLogs (filePath*)</code></li>
+  <li><code>/exim/action/importHar (filePath*)</code></li>
+  <li><code>/exim/action/importModsec2Logs (filePath*)</code></li>
+  <li><code>/exim/action/importUrls (filePath*)</code></li>
+  <li><code>/exim/action/importZapLogs (filePath*)</code></li>
+  <li>---</li>
+  <li><code>/exim/other/messageHar (id*)</code></li>
+  <li><code>/exim/other/messagesHar (baseurl start count)</code></li>
+  <li><code>/exim/other/messagesHarById (ids*)</code></li>
```

Comment:
Maybe should be called `export`?

Reply:
You mean ex: exportMessageHar?

Reply:
Or just `exportHar` if we merge them all (`id` and `ids` is exactly the same logic, so those could be merged to a single one regardless).